runtime/vm/ValueTypeHelpers.cpp

```diff
@@ -250,4 +250,15 @@ areValueTypesEnabled(J9JavaVM *vm)
 #endif /* defined(J9VM_OPT_VALHALLA_VALUE_TYPES) */
 }
 
+j9object_t
+getDefaultValue(J9Class* clz)
+{
+	Assert_VM_notNull(clz);
+	Assert_VM_true(clz->initializeStatus == J9ClassInitSucceeded);
```

Comment:
Please add a comment to explain the reason of this assertion. i.e. This helper is added for JIT and JIT will always call this helper after the class is initialized.

Reply:
added in e0724c2